src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v1/common.kt

```diff
@@ -159,129 +154,7 @@ fun refundTransaction(
-            !is RefundNotAllowedException ->
-              refundRetryService.enqueueRetryEvent(it, retryCount, tracingInfo)
-            else -> Mono.empty()
-          }
-        }
-        .thenReturn(tx)
-    }
-}
-
-fun handleVposRefundResponse(
-  transaction: BaseTransactionWithRequestedAuthorization,
-  refundResponse: VposDeleteResponseDto,
-  transactionsEventStoreRepository: TransactionsEventStoreRepository<TransactionRefundedData>,
-  transactionsViewRepository: TransactionsViewRepository
-): Mono<BaseTransaction> {
-  logger.info(
-    "Transaction requestRefund for transaction ${transaction.transactionId} PGS refund status [${refundResponse.status}]")
-
-  return when (refundResponse.status) {
-    StatusEnum.CANCELLED -> {
-      logger.info(
-        "Refund for transaction with id: [${transaction.transactionId.value()}] processed successfully")
-      updateTransactionToRefunded(
-        transaction, transactionsEventStoreRepository, transactionsViewRepository)
-    }
-    StatusEnum.DENIED -> {
-      logger.info(
-        "Refund for transaction with id: [${transaction.transactionId.value()}] denied with pgs response status: [${refundResponse.status}]! No more attempts will be performed")
-      updateTransactionToRefundError(
-        transaction, transactionsEventStoreRepository, transactionsViewRepository)
-    }
-    else ->
-      Mono.error(
-        RuntimeException(
-          "Refund error for transaction ${transaction.transactionId} unhandled PGS response status [${refundResponse.status}]"))
-  }
-}
-
-fun handleXpayRefundResponse(
-  transaction: BaseTransactionWithRequestedAuthorization,
-  refundResponse: XPayRefundResponse200Dto,
-  transactionsEventStoreRepository: TransactionsEventStoreRepository<TransactionRefundedData>,
-  transactionsViewRepository: TransactionsViewRepository
-): Mono<BaseTransaction> {
-  logger.info(
-    "Transaction requestRefund for transaction ${transaction.transactionId} PGS response status [${refundResponse.status}]")
-
-  return when (refundResponse.status) {
-    XPayRefundResponse200Dto.StatusEnum.CANCELLED -> {
-      logger.info(
-        "Refund for transaction with id: [${transaction.transactionId.value()}] processed successfully")
-      updateTransactionToRefunded(
-        transaction, transactionsEventStoreRepository, transactionsViewRepository)
-    }
-    XPayRefundResponse200Dto.StatusEnum.DENIED -> {
-      logger.info(
-        "Refund for transaction with id: [${transaction.transactionId.value()}] denied with pgs response status: [${refundResponse.status}]!! No more attempts will be performed")
-      updateTransactionToRefundError(
-        transaction, transactionsEventStoreRepository, transactionsViewRepository)
-    }
-    else ->
-      Mono.error(
-        RuntimeException(
-          "Refund error for transaction ${transaction.transactionId} unhandled PGS response status [${refundResponse.status}]"))
-  }
+  return Mono.empty()
```

Comment:
i 
```suggestion
  return Mono.error(RuntimeException("PGS refund not handled"))
```

or any other significant error message